### DISCUSSION TITLE
Job Aborted Error

### DISCUSSION THREAD

**Discussion by Keerthi1863** [pyiron/pyiron#1502] [Q&A]
![27](https://github.com/pyiron/pyiron/assets/139526741/7274c947-8246-4c47-8ca6-89e3f2fac9fb)
![28](https://github.com/pyiron/pyiron/assets/139526741/f7811ece-3144-4a87-b73f-1a95eeab56e0)
### Job getting aborted while running

**Comment by niklassiemer** [comment]
Welcome @Keerthi1863!

pyiron keeps track of previous runs / run attempts and reloads them if possible. Thus, I assume you tried to run that job previously and it failed. If you want to run it again specify `delete_existing_job=True` like suggested.

**Comment by Keerthi1863** [comment]
Dear Sir,

Thank you for your swift response. As per your suggestion, I changed  
the delete_existing_job = True and ran the code. Although the previous  
error 'delete_existing_job = False is meaningless' got rectified, a  
new error surfaced. The screen shot of the same has been attached with  
this mail. Kindly go through it and help us fix this error.

Thanking You,
Regards,
Keerthivasan

Quoting Niklas Siemer ***@***.***>:

**Reply by niklassiemer** [reply]
I fear the screen of the error is missing.

**Comment by Keerthi1863** [comment]
Sorry sir the error was a page long, now I'm sending the complete  
error. Kindly go thorough it.
Quoting Niklas Siemer ***@***.***>:

**Reply by niklassiemer** [reply]
I suppose, you are answering via email? The attachments don't seem to be added to the replies on github. Could you provide the error directly on github? Thanks

**Reply by Keerthi1863** [reply]
![30](https://github.com/pyiron/pyiron/assets/139526741/5521a5bd-b3ad-4b88-9633-a4a73585e38c)
![31](https://github.com/pyiron/pyiron/assets/139526741/ff685a22-43bd-4f27-8b87-00c76f35540d)
![32](https://github.com/pyiron/pyiron/assets/139526741/15bce7b7-01cc-4ae3-a604-b224163ff1fe)
![33](https://github.com/pyiron/pyiron/assets/139526741/d176257e-0db9-4be3-af0f-7548c21f8730)
![34](https://github.com/pyiron/pyiron/assets/139526741/c5e01b9f-0d8d-400b-b774-29a585b2b82e)

**Reply by pmrv** [reply]
Consult the files `error.log` and `error.msg` in the working directory of the job (`job.working_directory` or just `job['error.log']` and `job['error.msg']`).

**Reply by niklassiemer** [reply]
Up there is a warning that `lammps_serial.exe` could not be found. Is lammps properly installed?

**Reply by Keerthi1863** [reply]
I guess it is installed properly.. can you tell the procedure to install lammps

**Reply by niklassiemer** [reply]
The easiest is probably via conda- I assume you installed pyiron that way as well? Due to the `.exe` suffix I further assume you are working on native Windows (consider using the Linux subsystem for Windows - a lot of simulation programs are not available for Windows). In these cases it should be possible to install lammps via
```
conda install -c conda-forge pyiron lammps
```
if I recall this correctly right now.

**Reply by niklassiemer** [reply]
@pmrv could you quickly cross check xD I am on vacation today and cannot check right now :)

**Reply by pmrv** [reply]
I've edited the command, but it's mostly right.   If you are using the WSL then the above should set up everything correctly.  If you are on native windows, double check that:
1. lammps is available by trying to run `lmp_serial.exe` on the commandline.  If you have built lammps manually for windows the exact name and installation location can vary and we can't help you with that.
2. once you know lammps works on its own, check that the job script uses it correctly.  For that find out where the script is stored `job.executable`.  This should point to a text file.  Edit this file to use the correct invocation of lammps that you have determined in step 1.

**Reply by niklassiemer** [reply]
Well I tend to remwmber there was a lammps build for Windows on the pyiron chanel, was it not?

**Reply by jan-janssen** [reply]
> Well I tend to remwmber there was a lammps build for Windows on the pyiron chanel, was it not?

```
conda install -c pyiron lammps
```
https://anaconda.org/pyiron/lammps

**Reply by Keerthi1863** [reply]
![photo_2023-07-16_02-39-39](https://github.com/pyiron/pyiron/assets/139526741/56598588-561d-4f5c-b4fd-a0b472c5ed75)
installed anaconda in linux and we tried installing pyiron using the codes given in the site but we are getting this unsuccesful attempt command as output for all the 4 codes.
linux is new to us so can you suggest a method or video to install pyiron properly.

**Reply by Keerthi1863** [reply]
thank you very much the error got sorted out. newer version has some problems i guess that's why we couldn't proceed further. older(2018) version lammps works, and can you please provide us the link for sphinx and calphy too.

**Reply by jan-janssen** [reply]
The sphinx DFT code is unfortunately only available for Linux, so to run a sphinx DFT simulation on windows you need the windows subsystem for Linux (WSL) and reinstall the Conda environment there. For calphy you can use:
```
conda install -c conda-forge calphy
```
https://anaconda.org/conda-forge/calphy

**Reply by Keerthi1863** [reply]
thank you sir.
